dallinger/recruiters.py

```diff
@@ -1523,6 +1511,26 @@ def get_qualifications(self, hit_id, sandbox):
         service = self.load_service(sandbox)
         return service.get_study(hit_id)["QualificationRequirements"]
 
+    def validate_config(
+        self, mode_from_command=None, open_recruitment_from_command=False
```

Comment:
To make this future-proof I think it would be safest to do something like 

```py
def validate_config(self, cli_invocation: dict):
    cli_invocation["open_recruitment"]
    cli_invocation["mode"] == "live"
    ...
```

Reply:
:+1:  Now using `cli_invocation_options` dict as single param